client/jetpack-connect/jetpack-connect-notices.jsx

```diff
@@ -183,9 +187,24 @@ export class JetpackConnectNotices extends Component {
 				noticeValues.status = 'is-warning';
 				noticeValues.icon = 'notice';
 				return noticeValues;
+
+			case XMLRPC_ERROR:
+				noticeValues.text = translate( 'We had trouble connecting.' );
+				return noticeValues;
 		}
 	}
 
+	getNoticeActionText() {
+		const { noticeType, translate } = this.props;
```

Comment:
Any particular reason not to do this in the `getNoticeValues()` switch with something like
```
noticeValues.actionText = translate( 'Try again' );
```
?

Reply:
Yeah - I didn't want to further play with that object, because currently we're passing all of these to the `<Notice />` component:

```
<Notice { ...noticeValues } />
```

and we would need to further `pickBy/omit` some of them and pass them to the `<NoticeAction />` - which sounds far from ideal. I preferred the cleaner approach of separating the notice and its action (considering that we might actually not have a notice action in most of the cases).